tests/common/snappi/snappi_fixtures.py

```diff
@@ -517,3 +522,371 @@ def cvg_api(snappi_api_serv_ip,
+                peer_ports = [linecards[line_card][asic] for line_card in linecards.keys()]
+                peer_ports = list(zip(*peer_ports))
+                peer_ports = [item for sublist in peer_ports for item in sublist]
+                return peer_ports[:number_of_tgen_peer_ports]
+        else:
+            raise Exception('Error: Not enough line_card_choice')
+
+    elif line_card_choice in ['chassis_multi_line_card_multi_asic']:
+        # Different line card and minimum one port from different asic number
+        if len(linecards.keys()) >= 2:
+            host_asic = list(zip(config_set[line_card_choice]['hostname'], config_set[line_card_choice]['asic']))
+            peer_ports = list(zip_longest(*[linecards[host][asic] for host, asic in host_asic]))
+            peer_ports = [item for sublist in peer_ports for item in sublist]
+            peer_ports = list(filter(None, peer_ports))
+            return peer_ports[:number_of_tgen_peer_ports]
+        else:
+            raise Exception('Error: Not enough line_card_choice')
+
+
+def create_ip_list(value, count, mask=32, incr=0):
+    '''
+        Create a list of ips based on the count provided
+        Parameters:
+            value: start value of the list
+            count: number of ips required
+            mask: subnet mask for the ips to be created
+            incr: increment value of the ip
+    '''
+    if sys.version_info.major == 2:
+        value = unicode(value)          # noqa: F821
+
+    ip_list = [value]
+    for i in range(1, count):
+        if ip_address(value).version == 4:
+            incr1 = pow(2, (32 - int(mask))) + incr
+            value = (IPv4Address(value) + incr1).compressed
+        elif ip_address(value).version == 6:
+            if mask == 32:
+                mask = 64
+            incr1 = pow(2, (128 - int(mask))) + incr
+            value = (IPv6Address(value) + incr1).compressed
+        ip_list.append(value)
+
+    return ip_list
+
+
+def cleanup_config(duthost_list, snappi_ports):
+    for index, duthost in enumerate(duthost_list):
+        port_count = len(snappi_ports)
+        dutIps = create_ip_list(dut_ip_start, port_count, mask=prefix_length)
+        for port in snappi_ports:
+            if port['peer_device'] == duthost.hostname:
+                port_id = port['port_id']
+                dutIp = dutIps[port_id]
+                logger.info('Removing Configuration on Dut: {} with port {} with ip :{}/{}'.format(
+                                                                                                   duthost.hostname,
+                                                                                                   port['peer_port'],
+                                                                                                   dutIp,
+                                                                                                   prefix_length))
+                if port['asic_value'] == 'None':
```

Comment:
if port['asic_value'] == 'None' -- wrong
It must be without quotes
if port['asic_value'] == None -- correct